docs/rtk-query/usage/pagination.mdx

```diff
@@ -76,6 +76,43 @@ const PostList = () => {
 };
 ```
 
+## Pagination and Caching
+
+If you have caching enabled for your paginated queries, it maybe a good idea to invalidate the `LIST` cache as well when you do a `delete` mutation.
+
+```ts title="Example of invalidating cache for paginated queries"
+// file: types.ts noEmit
+export interface Post {
+  id: number
+  name: string
+}
+
+// file: api.ts
+// Or from '@reduxjs/toolkit/query/react'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+import { Post } from './types'
+
+export const postApi = createApi({  
+  reducerPath: 'postsApi',  
+  baseQuery: fetchBaseQuery({ baseUrl: '/' }),  
+  tagTypes: ['Posts'],  
+  endpoints: (build) => ({
+    deletePost: build.mutation<{ success: boolean; id: number }, number>({ 
+      query(id) {        
+        return {          
+          url: `post/${id}`,          
+          method: 'DELETE',        
+        }      
+      },      
+      // Invalidates queries that subscribe to this Post `id` as well as the `LIST`.
+      invalidatesTags: (result, error, id) => [
+        { type: "Posts", id },
+        { type: "Posts", id: "LIST" },
+      ],
+    }),
+  }),
+})
```

Comment:
```suggestion
// file: api.ts
import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
interface Post {
  id: number
  name: string
}
interface ListResponse<T> {
  page: number
  per_page: number
  total: number
  total_pages: number
  data: T[]
}

export const postApi = createApi({
  reducerPath: 'postsApi',
  baseQuery: fetchBaseQuery({ baseUrl: '/' }),
  tagTypes: ['Posts'],
  endpoints: (build) => ({
    listPosts: build.query<ListResponse<Post>, number | void>({
      query: (page = 1) => `posts?page=${page}`,
      // highlight-start
      providesTags: (result, error, page) =>
        result
          ? [
              // Provides a tag for each post in the current page,
              // as well as the 'PARTIAL-LIST' tag.
              ...result.data.map(({ id }) => ({ type: 'Posts' as const, id })),
              { type: 'Posts', id: 'PARTIAL-LIST' },
            ]
          : [{ type: 'Posts', id: 'PARTIAL-LIST' }],
      // highlight-end
    }),
    deletePost: build.mutation<{ success: boolean; id: number }, number>({
      query(id) {
        return {
          url: `post/${id}`,
          method: 'DELETE',
        }
      },
      // Invalidates the tag for this Post `id`, as well as the `PARTIAL-LIST` tag,
      // causing the `listPosts` query to re-fetch if a component is subscribed to the query.
      // highlight-start
      invalidatesTags: (result, error, id) => [
        { type: 'Posts', id },
        { type: 'Posts', id: 'PARTIAL-LIST' },
      ],
      // highlight-end
    }),
  }),
})
```

This changes a few things:
- Shows the types explicitly
- Includes the `listPosts` paginated query and the tags it provides (no point invalidating tags when there is no other query providing them)
- Uses the `PARTIAL-LIST` tag for clarity
- Removes the "// Or from '@reduxjs/toolkit/query/react'" comment, which is already the same import, and not really necessary to include for this section of the docs
- Makes the tag providing/invalidating more obvious